5.process_CP_features/scripts/6.aggregation.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Perform feature selection on normalized data
+
+# ## Import libraries
+
+# In[1]:
+
+
+import gc
+import pathlib
+
+import pandas as pd
+from pycytominer import aggregate
+
+# ## Set paths and variables
+
+# In[2]:
+
+
+# set paths
+paths_dict = {
+    "timelapse_profiles": {
+        "input_dir": pathlib.Path(
+            "../data/5.feature_select/profiles/features_selected_profile.parquet"
+        ).resolve(strict=True),
+        "output_file_dir": pathlib.Path(
+            "../data/6.aggregated/profiles/aggregated_profile.parquet"
+        ).resolve(),
+    },
+    "endpoint_data": {
+        "input_dir": pathlib.Path(
+            "../data/5.feature_select/endpoints/features_selected_profile.parquet"
+        ).resolve(strict=True),
+        "output_file_dir": pathlib.Path(
+            "../data/6.aggregated/endpoints/aggregated_profile.parquet"
+        ).resolve(),
+    },
+}
+
+
+# ## Perform feature selection
+
+# In[3]:
+
+
+# define operations to be performed on the data
+# list of operations for feature select function to use on input profile
+feature_select_ops = [
+    "variance_threshold",
+    "blocklist",
+    "drop_na_columns",
+    "correlation_threshold",
+]
+
+
+# In[4]:
+
+
+manual_block_list = [
+    "Nuclei_AreaShape_BoundingBoxArea",
+    "Nuclei_AreaShape_BoundingBoxMinimum_X",
+    "Cells_AreaShape_BoundingBoxArea",
+]
+
```

Comment:
I don't see feature selection performed again in this script, recommend removing.